### PR TITLE
new bash exercise: all-your-base

### PR DESCRIPTION
**Pull request by glennj**
Hi, my first new exercise. Assuming this goes well, I'd like to contribute a bunch more bash exercises.

[canonical-data.json](https://github.com/guygastineau/problem-specifications/blob/master/exercises/all-your-base/canonical-data.json)

---

Reviewer Resources:

[Track Policies](https://github.com/exercism/bash/blob/master/POLICIES.md)

### CODE REVIEW

**Comment by guygastineau** [comment]
Sorry no one has responded yet.

I will check this out today.  Thank you for your work.

File: exercises/all-your-base/all_your_base_test.sh [hunk 1]
```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env bash
+
+@test 'single_bit_to_one_decimal' {
```
Comment (guygastineau) [inline exercises/all-your-base/all_your_base_test.sh]: add a commented `#skip` here.

File: exercises/all-your-base/all_your_base_test.sh [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+    run bash all_your_base.sh 2 "1" 10
+    [ "$status" -eq 0 ]
+    [ "$output" == "1" ]
+}
+
+@test 'binary_to_single_decimal' {
```
Comment (guygastineau) [inline exercises/all-your-base/all_your_base_test.sh]: From here onward add an uncommented `skip` to each exercise.

File: exercises/all-your-base/all_your_base_test.sh [hunk 3]
```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env bash
+
+@test 'single_bit_to_one_decimal' {
+    run bash all_your_base.sh 2 "1" 10
+    [ "$status" -eq 0 ]
```
Comment (guygastineau) [inline exercises/all-your-base/all_your_base_test.sh]: We are supposed to be switching from `[` to `[[`

**Comment by guygastineau** [comment]
I meant for those to be requested changes, but I don't know how to change the status of a review, so please hold off on merging until the requests from the review have been satisfied.

**Comment by guygastineau** [comment]
@glennj 
Really great work on this.  Once the little details from my review are handled I will merge this.
I am also requesting another review, but I don't think there will be any issues.

Cheers.

**Comment by kotp** [comment]
> I meant for those to be requested changes, but I don't know how to change the status of a review, so please hold off on merging until the requests from the review have been satisfied.

I have reset the review for you @guygastineau

**Comment by guygastineau** [comment]
Once you are satisfied @kotp then I think we are good to go :smile:

**Comment by kotp** [comment]
Thank you @glennj and for the support @guygastineau

**Pull request merged**